go.mod

```diff
@@ -53,6 +54,7 @@ require (
 	github.com/jinzhu/now v1.1.5 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/linuxsuren/go-fake-runtime v0.0.4 // indirect
+	github.com/linuxsuren/oauth-hub v0.0.0-20240809060240-e78c21b5d8d4 // indirect
```

Comment:
Prefer to use version `v.0.0.1`. See https://github.com/LinuxSuRen/oauth-hub/releases/tag/v0.0.1